src/compressed_tensors/quantization/quant_scheme.py

```diff
@@ -199,6 +199,18 @@ def is_preset_scheme(name: str) -> bool:
     ),
 )
 
+# AWQ quantization
+AWQ = dict(
```

Comment:
Why is this needed?

Reply:
Not needed; just nice to have a preset, easier to define a scheme in QuantizationModifier than specifying QuantizationArgs over and over

Reply:
I talked to rahul about this, we can either add a `"W4A16_ASYMMETRIC"` preset or remove entirely. I am not a big fan of presets here, this seems like something we want the user to be explicit about. what do people think?

Reply:
In general I'm in factor of creating generic and composable schemes rather than creating paper-specific presets. I'd rather remove this

Reply:
Couldn't we just do this, and create an alias?
https://github.com/neuralmagic/compressed-tensors/blob/b762e5655ed5b425e30544c393d9762426c36288/src/compressed_tensors/quantization/quant_scheme.py#L210

I think W4A16_symmetric makes sense with an AWQ alias

Reply:
Moved to `W4A16_ASYM` preset, leaving off `"AWQ"` alias for now since it's orthogonal to AWQ, just what we've been using for lining up with the paper's table 4.